tests/kafkatest/services/security/minikdc.py

```diff
@@ -48,6 +49,16 @@ def start_node(self, node):
         self.logger.info("minikdc.properties")
         self.logger.info(props_file)
 
+        # create local temp dir where we will store keytab and krb5conf files
```

Comment:
@granders  Agreed that this looks cleaner. I will add that method and create a temp file with unique name (rather than the directory) in /tmp folder so that there is no conflict